here-naksha-lib-plv8/src/commonMain/kotlin/NakshaBulkLoaderOp.kt

```diff
@@ -60,7 +60,9 @@ internal class NakshaBulkLoaderOp(
                 row[COL_GEOMETRY] = geo_arr[i]
                 row[COL_FEATURE] = feature_arr[i]
                 row[COL_GEO_TYPE] = geo_type_arr[i]
-                row[COL_GEO_GRID] = opReader.grid()
+                if (opReader.grid() != null) {
```

Comment:
Why does the grid in the row must be undefined, shouldn't it be better to allow _null_? I do not see the need for this check, can't the value simply be _null_?

Reply:
When we need to return rows with values, we need to put the values from the request to existing modified rows, but if geo_grid is null in the request and we calculate it in writeFeatures then 'null' value will override calculated one.
I agree that this is not obvious and maybe I shouldn't blindly merge calculatedRow with request row. Merging every value by name would be more clear I think

Reply:
Okay, get it, can you just leave a small comment to explain it for the next asking himself about it? :D